### DISCUSSION TITLE
Welcome to aspoem Discussions!

### DISCUSSION THREAD

**Discussion by meetqy** [meetqy/aspoem#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by gavinqd** [comment]
failed to solve: rpc error: code = Unknown desc = failed to solve with frontend dockerfile.v0: failed to read dockerfile: open /volume1/@docker/tmp/buildkit-mount3035347666/Dockerfile: no such file or directory
大佬，请问这个错误该怎么解决呀，我把Dockerfile.txt和docker-compose.yaml文件都放到docker/aspoem文件夹下对吗？
